cmake/FindLIBZIP.cmake

```diff
@@ -0,0 +1,30 @@
+find_package(ZLIB REQUIRED)
+find_package(BZip2 REQUIRED)
+
+find_path(LIBZIP_INCLUDE_DIR NAMES zip.h)
+mark_as_advanced(LIBZIP_INCLUDE_DIR)
+
+find_library(LIBZIP_LIBRARY NAMES zip)
+mark_as_advanced(LIBZIP_LIBRARY)
+
+include(CMakeFindDependencyMacro)
+find_package_handle_standard_args(
+    LIBZIP 
+    REQUIRED_VARS
+        LIBZIP_LIBRARY
+        LIBZIP_INCLUDE_DIR
+)
+
+if (LIBZIP_FOUND)
+    set(LIBZIP_INCLUDE_DIRS "${LIBZIP_INCLUDE_DIR}")
+
+    if (NOT TARGET libzip::libzip)
+        add_library(libzip::libzip UNKNOWN IMPORTED)
+        set_target_properties(libzip::libzip
+            PROPERTIES
+                INTERFACE_INCLUDE_DIRECTORIES ${LIBZIP_INCLUDE_DIRS}
```

Comment:
Okay, I managed to make it work, at least for shared library. In order to work, I had to also quote `"${LIBZIP_INCLUDE_DIRS}"` and then, the following compiles:
```
make libraries
mkdir build
cd build
cmake -D LIBZIP_INCLUDE_DIR="../lib/libzip-1.5.2/lib/;../lib/libzip-1.5.2/build/" -D LIBZIP_LIBRARY=../lib/libzip-1.5.2/build/lib/libzip.so ..
make
```

Can you please make this change and confirm my commands are correct ? Thanks !

Reply:
Ping @vicroms :sweat_smile: Is it possible to have a feedback any soon ? Thanks !

Reply:
Hi @ctabin 

Sorry, I've been a bit busy lately. 
A better approach would be to add an option that can be passed via the command line to give CMake paths where to look for `libzip`.

Kind of like what built-in modules do: 

[FindZLIB.cmake](https://cmake.org/cmake/help/v3.0/module/FindZLIB.html)

> An includer may set ZLIB_ROOT to a zlib installation root to tell this module where to look.

Reply:
OK thanks for the insight :)